src/common/SPELLS/PRIEST.js

```diff
@@ -446,6 +446,30 @@ export default {
     icon: 'spell_holy_guardianspirit',
   },
 
+  HOLY_PRIEST_T21_2SET_BONUS_BUFF: {
+    id: 251831,
+    name: 'Holy Priest T21 2P Bonus',
+    icon: 'ability_priest_ascension',
+  },
+
+ HOLY_PRIEST_T21_2SET_BUFF: {
+    id: 253437,
+    name: 'Holy Priest T21 2P Bonus',
```

Comment:
Please use the spell's actual name, "Answered Prayers" for both the name field and the whole var's name. This will also make it easier to tell apart from the other buff here. If you're worried about a future dev being confused about what this is, add a comment like `// proc from Holy Priest T21 2set`